source/agora/stats/EndpointReqestStats.d

```diff
@@ -0,0 +1,16 @@
+module agora.stats.EndpointReqestStats;
+
+import agora.stats.Stats;
+
+public struct EndpointRequestStatsLabel
+{
+    string endpoint;
+    string protocol;
+}
+
+public struct EndpointRequestStatsValue
+{
+    ulong agora_endpoint_calls_total;
+}
```

Comment:
If that's an aggregation, that is better left to upstream, as discussed in the other comment